plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/toolwindow/AmazonQToolWindow.kt

```diff
@@ -25,8 +24,7 @@ class AmazonQToolWindow private constructor(
     val component
         get() = chatPanel.component
 
-    fun disposeAndRecreate() {
-        Disposer.dispose(chatPanel)
```

Comment:
not sure what will happen if we keep creating new panel and they are never disposed, will it cause memory leak

Reply:
yes we would need to dispose if we are creating new panels